### PR TITLE
[#2187] Remove `eIC` reference from the marketplace

### LINKED ISSUE #2187: remove the 'eIC' reference from the MP
from both presentation and DB layer.
Replace wirt "EOSCRegistry"

### PR DESCRIPTION
**Pull request by goreck888**
Change all references, labels and names of variables containing `eIC`
to the `EOSC Registry` reference excluding changelog.
Closes #2187

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Please view this file on the master branch, on stable branches it's out of date.
 - Enhance the Ordering API OMS trigger (@jswk)
 - API authorization token can now only be revoked by regenerating it (@jswk)
 - `api_docs` page is available for unlogged user (@goreck888)
+- All references to `eIC` are removed and changed to the `EOSC Registry` (@goreck888)
```
Comment (kmarszalek) [inline CHANGELOG.md:34]: removed or changed?

**Review by Marcelinna — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (goreck888) [reply CHANGELOG.md]: Changed :smile:

File: docs/upgrade/3.16.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Upgrade to 3.16.0
```
Comment (jswk) [inline docs/upgrade/3.16.0.md:1]: We're skipping 3.16.0, so the upgrade file should be named and titled `3.17.0(.md)`.

Comment (goreck888) [reply docs/upgrade/3.16.0.md:1]: Ok.

**Review by jswk — APPROVED** [review]

**Pull request merged**